Source/Eos/EOS.H

```diff
@@ -20,6 +24,8 @@ using EosType = eos::GammaLaw;
 using EosType = eos::Fuego;
 #elif USE_SRK_EOS
 using EosType = eos::SRK;
+#elif USE_MANIFOLD_EOS && !defined(AMREX_USE_SYCL)
```

Comment:
are there other abort elsewhere that disallow sycl for manifold? like in the gnu make or cmake paths?

Reply:
This should catch the condition regardless of build system and PelePhysics/C/LMeX. I think it would take too many checks to do it in both build systems for all application codes.